pkgs/top-level/aliases.nix

```diff
@@ -246,6 +246,7 @@ mapAliases ({
   libtidy = html-tidy;  # added 2014-12-21
   libudev = udev; # added 2018-04-25
   libsexy = throw "libsexy has been removed from nixpkgs, as it's abandoned and no package needed it."; # 2019-12-10
+  libqmatrixclient = throw "libqmatrixclient was renamed to libQuotient"; # added 2020-04-09
```

Comment:
are `libqmatrixclient` and `libquotient` no longer api/abi comparable?
I'm asking because aliases are for renames, and an alias by definition would be `libqmatrixclient = libquotient`.

Reply:
The name of the actual library has changed something that expects `libqmatrixclient` will not work with `libquotient`.

Reply:
> The name of the actual library has changed something that expects `libqmatrixclient` will not work with `libquotient`.

Yep, that makes sense.

Reply:
```suggestion
  libqmatrixclient = throw "libqmatrixclient was renamed to libquotient"; # added 2020-04-09
```